include/pros/apix.h

```diff
@@ -550,7 +666,110 @@ int32_t fdctl(int file, const uint32_t action, void* const extra_arg);
  */
 #define DEVCTL_SET_BAUDRATE 17
 
+
+/******************************************************************************/
+/**                       RTOS STATIC TASK FACILITIES                        **/
+/******************************************************************************/
+
+#ifdef __cplusplus
+#define task_t pros::task_t
+#define task_fn_t pros::task_fn_t
+#define mutex_t pros::mutex_t
+#define sem_t pros::c::sem_t
+#define queue_t pros::c::queue_t
+#endif
+
+typedef uint32_t task_stack_t;
+
+/**
+ * Creates a task using statically allocated buffers. All tasks used by the PROS
+ * system must use statically allocated buffers.
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENOMEM - The stack cannot be used as the TCB was not created.
+ *
+ * \param function
+ *        Pointer to the task entry function
+ * \param parameters
+ *        Pointer to memory that will be used as a parameter for the task being
+ *        created. This memory should not typically come from stack, but rather
+ *        from dynamically (i.e., malloc'd) or statically allocated memory.
+ * \param prio
+ *        The priority at which the task should run.
+ *        TASK_PRIO_DEFAULT plus/minus 1 or 2 is typically used.
+ * \param stack_depth
+ *        The number of words (i.e. 4 * stack_depth) available on the task's
+ *        stack. TASK_STACK_DEPTH_DEFAULT is typically sufficienct.
+ * \param name
+ *        A descriptive name for the task.  This is mainly used to facilitate
+ *        debugging. The name may be up to 32 characters long.
+ *
```

Comment:
You're missing docs for stack_buffer and task_buffer here.

Reply:
Fixed at cb96b0c